contracts/Dispatcher.sol

```diff
@@ -657,6 +635,7 @@ contract Dispatcher is IbcDispatcher, Ownable {
         require(!hasAckPacketCommitment, 'Ack packet commitment already exists');
         ackPacketCommitment[address(receiver)][packet.dest.channelId][packet.sequence] = true;
 
+        emit RecvPacket(address(receiver), packet.dest.channelId, packet.sequence);
```

Comment:
This is the duplicate logic for line 611, so we will see duplicate `RecvPacket` events as receipts.